package.json

```diff
@@ -48,7 +48,7 @@
     "react-popper": "^2.3.0"
   },
   "dependencies": {
-    "@flowplayer/player": "3.2.8",
+    "@meemoo/react-components": "^2.21.1",
```

Comment:
this should probably be a peer dependency since we ultimately want to get a single version of the react-components in the client js bundle
it could be that you'll need to mark @meemoo/react-components as an external in the rollup config file: rollup.config.js

Reply:
I've marked it as a peer & dev dependency to ensure the codebase can be built; implementing projects shouldn't pull the dev dependencies as a result.